sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -361,6 +361,7 @@ object FunctionRegistry {
     expression[Ceil]("ceil"),
     expression[Ceil]("ceiling", true),
     expression[Cos]("cos"),
+    expression[Sec]("sec"),
```

Comment:
Whenever you add a new function, please run `ExpressionSchemaSuite` and re-generate golden file.
See also the head of `ExpressionSchemaSuite.scala`.

Reply:
Thank you, I just pushed it.